packages/nimble/inst/include/nimble/Utils.h

```diff
@@ -165,6 +165,8 @@ void nimStop();
 
 bool nimNot(bool x);
 
+bool isTRUE(bool x);
```

Comment:
@paciorek I suggest in the .h:
static inline bool isTRUE(bool x) {return(x);}
and then nothing is needed in the .cpp and the compiler should inline the call.